src/app/plugins/garage.plugin.ts

```diff
@@ -0,0 +1,86 @@
+import { Plugin } from '../../common/plugin';
+import { IContainer, IMessage, ChannelType, IHttpResponse } from '../../common/types';
+
+const cheerio = require("cheerio")
+
+class Garage {
+	public name : string = "";
+	public available: number = 0;
+	public saturation: number = 0;
+	public capacity: number = 0;
+	public percent_full: number = 0;
+	public percent_avail: number = 0;
+}
+
+export class GaragePlugin extends Plugin {
+	public name: string = 'Garage Plugin';
+	public description: string = 'Get garage status';
+	public usage: string = 'garage [which garage]';
+	public permission: ChannelType = ChannelType.Public;
+
+	private _API_URL: string = "http://secure.parking.ucf.edu/GarageCount/iframe.aspx";
+	private title_message: string = "**Current UCF Garage Saturation**"
```

Comment:
The pattern with private variables/methods in this project is always start with a leading `_`. We also use semicolons throughout the codebase. Let's keep consistent to that.